stingray/lightcurve.py

```diff
@@ -595,3 +596,37 @@ def plot(self, labels=None, axis=None, title=None, marker='-', save=False,
                 plt.savefig('out.png')
             else:
                 plt.savefig(filename)
+
+    def save(self, filename = 'lc.pickle', format = 'pickle', save_as_dict = False):
```

Comment:
I'd probably make `filename` a required parameter without default? Probably in practice anyone saving a light curve to file will want to pass it the desired file name.